### PR TITLE
meta(codeowners): Add js build owners

### PR DESCRIPTION
**Pull request by billyvg**
Adds github codeowners for js-build-related files (webpack config, tsconfig, etc)

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -32,9 +32,14 @@
 /docker             @getsentry/releases
 setup.py            @getsentry/releases
 setup.cfg           @getsentry/releases
-# TODO: owners-js-build, owners-build (for travis / gha)
+# TODO: owners-build (for travis / gha)
 requirements*.txt   @getsentry/owners-python-build
 pyproject.toml      @getsentry/owners-python-build
+tsconfig.*          @getsentry/owners-jsbuild
```
Comment (joshuarli) [inline .github/CODEOWNERS:38]: Can you rename to `owners-js-build`? For consistency.

Comment (EvanPurkhiser) [reply .github/CODEOWNERS:38]: I'll see if I can get @dcramer to do real fast

Comment (EvanPurkhiser) [reply .github/CODEOWNERS:38]: done https://github.com/orgs/getsentry/teams/owners-js-build

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -32,9 +32,14 @@
...
+webpack.config.*    @getsentry/owners-jsbuild
+package.json        @getsentry/owners-jsbuild
+babel.config.*      @getsentry/owners-jsbuild
+build-utils/        @getsentry/owners-jsbuild
```
Comment (billyvg) [inline .github/CODEOWNERS:42]:
```suggestion
tsconfig.*          @getsentry/owners-js-build
webpack.config.*    @getsentry/owners-js-build
package.json        @getsentry/owners-js-build
babel.config.*      @getsentry/owners-js-build
build-utils/        @getsentry/owners-js-build
```

**Review by joshuarli — APPROVED** [review]
Great, thanks for adding more codeowners coverage!

**Pull request merged**